### PR TITLE
prep-Javascript-variables Nathan Chon

### PR DESCRIPTION
**Pull request by Nathan-Chon**
Screenshot_20230115_092312

### CODE REVIEW

File: prep-javascript-variables/index.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+<html>
+<head>
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    var fullName = 'Nathan Chon';
+    console.log(fullName);
+    var NumberOfPets = 1;
```
Comment (BrettAlbright) [inline prep-javascript-variables/index.html]: only special variables should start with capital letters. I would change this to a lowercase n

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update needed

**Comment by Nathan-Chon** [comment]
Screenshot_20230116_125706
fixed capitalization problem

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**